### PR TITLE
UI fixes for Mac and Linux

### PR DESCRIPTION
**Pull request by cpaulgilman**
* UI form fixes for Mac and Linux
* Set width of tech list in configuration window and navigation menu in main window so long labels in collapsible trees are visible. This is a workaround, better solution would be to size tree width based on longest collapsed label.

### CODE REVIEW

File: deploy/runtime/ui/Fuel Cell Costs Hybrid.json [hunk 1]
```diff
@@ -2771,7 +2771,7 @@
                 },
                 "Caption": {
                     "Type": 5.0,
-                    "String": "The total installed cost is the sum of the indirect, sales tax, and direct costs. Note that it does not include any financing costs from the Financial Parameters page."
+                    "String": "See the Hybrid Cost page for the total installed cost of the hybrid system. The total installed cost of the wind subsystem is the sum of the indirect, sales tax, and direct costs. Note that it does not include any financing costs from the Financial Parameters page."
```
Comment (mjprilliman) [inline deploy/runtime/ui/Fuel Cell Costs Hybrid.json:2774]: fuel cell subsystem not wind.

File: deploy/runtime/ui/Generic System Costs.json [hunk 2]
```diff
@@ -3531,11 +3531,21 @@
...
-        "//\tfin = value(\"case.financing\");",
-        "//\t",
-        "//\tshow( \"lblFinancingCost\", ((fin!=0)&&(fin!=1)&&(fin!=2)));",
-        "//};"
+        "on_load{'Generic System Costs'} = define() {",
+        "\ttech = technology();",
+        "\tif ( tech == 'Generic PVWatts Wind FuelCell Battery Hybrid' )",
+        "\t{",
+        "\t\tgs_total_cost_name = 'Total Installed Cost of Generic Subsystem';",
+        "\t\tgs_total_cost_description = 'See the Hybrid Cost page for the total installed cost of the hybrid system. The total installed cost of the Generic subsystem is the sum of the indirect, sales tax, and direct costs. Note that it does not include any financing costs from the Financial Parameters page.';",
```
Comment (mjprilliman) [inline deploy/runtime/ui/Generic System Costs.json:3539]: Generic and PV are capitalized, so maybe capitalize Wind on the wind cost page?

File: deploy/runtime/ui/MSLF System Control.json [hunk 3]
```diff
@@ -1354,7 +1354,7 @@
                 },
                 "X": {
                     "Type": 3.0,
-                    "Integer": 258.0
+                    "Integer": 300.0
```
Comment (mjprilliman) [inline deploy/runtime/ui/MSLF System Control.json:1357]: Not a big deal but different width of group boxes: 
image

File: deploy/runtime/ui/Operating Costs Land Lease.json [hunk 4]
```diff
@@ -463,19 +463,19 @@
                 },
                 "X": {
                     "Type": 3.0,
-                    "Integer": 276.0
+                    "Integer": 273.0
```
Comment (mjprilliman) [inline deploy/runtime/ui/Operating Costs Land Lease.json:466]: image
Not sure if this will clip outside of Windows?

**Review by mjprilliman — APPROVED** [review]
Looks good, just a few nitpicks

**Review by sjanzou — APPROVED** [review]
Smaller font looks good on the Mac
image

The funky scrolling through the top ribbon on the mac may be a separate issue
image

Forms look good
image

**Pull request merged**